frontend/packages/kubevirt-plugin/src/selectors/event/filters.ts

```diff
@@ -10,18 +10,33 @@ import {
 import { VIRT_LAUNCHER_POD_PREFIX } from '../../constants/vm';
 import { VMILikeEntityKind } from '../../types/vmLike';
 
-type EventFilterFunction = (src: EventInvolvedObject) => boolean;
+type EventInvolvedObjectWithTimestamp = EventInvolvedObject & { timestamp?: string };
+
+type EventFilterFunction = (src: EventInvolvedObjectWithTimestamp) => boolean;
```

Comment:
given what I wrote about passing whole event to filter function this will become
```suggestion
type EventFilterFunction = (src: EventInvolvedObject, event: EventKind) => boolean;
```

Reply:
done